drivers/starburst/src/metabase/driver/implementation/driver_helpers.clj

```diff
@@ -28,5 +28,5 @@
                               :native-parameters               true
                               :expression-aggregations         true
                               :binning                         true
-                              :foreign-keys                    true}]
+                              :foreign-keys                    false}]
```

Comment:
This line could be omitted. Seems most other adapters do that in metabase that don't support foreign keys.

Reply:
Maybe, but I will leave in to call out I removed it for future reference.